benefit/data/sheets/ComplexTypes/ComplexTypes.json

```diff
@@ -452,5 +452,7 @@
   {"LiveFrom": "18/06/2019", "ID": "otherPartyHearingRecordingReqUi", "ListElementCode": "hearingRecordingStatus", "FieldType": "DynamicList", "ElementLabel": "Status", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "18/12/2021", "ID": "OtherPartyOption", "ListElementCode": "otherPartyOptionName", "FieldType": "Text", "ElementLabel": "Other party name", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "18/12/2021", "ID": "OtherPartyOption", "ListElementCode": "otherPartyOptionId", "FieldType": "Text", "ElementLabel": "Other party id", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "18/12/2021", "ID": "OtherPartyOption", "ListElementCode": "resendToOtherParty", "FieldType": "YesOrNo", "ElementLabel": "Reissue", "SecurityClassification": "PUBLIC"}
+  {"LiveFrom": "18/12/2021", "ID": "OtherPartyOption", "ListElementCode": "resendToOtherParty", "FieldType": "YesOrNo", "ElementLabel": "Reissue", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "18/12/2021", "ID": "CaseManagementLocation", "ListElementCode": "region", "FieldType": "Text", "ElementLabel": "Region", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "18/12/2021", "ID": "CaseManagementLocation", "ListElementCode": "baseLocation", "FieldType": "Text", "ElementLabel": "Base Location", "SecurityClassification": "PUBLIC"}
```

Comment:
is uppercase C the correct thing for the id here?

Reply:
I see complex types have started with both uppercase letters as well as lowercase. I think upper case is more suitable as this is a new type. Anyway no any functional issues with this ID